src/main/resources/templates/team_page_application_submitted.html

```diff
@@ -17,7 +23,25 @@ <h3 th:text="#{teamPage.submitTeamApp.header}">Your application has been receive
                 </div> <!-- col -->
                 <div class="col-md-1"></div>
             </div> <!-- row -->
+            <div class="cc-wraper">
+                <div class="cc-contactpop fixed">
+                    <h1>Do you need Help? <span>Get in touch.</span></h1>
+                    <div class="renseignement">
+                        <i class="fa fa-times"></i>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)